src/core/Microsoft.PowerToys.Settings.UnitTest/ModelsTests/HelperTest.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.PowerToys.Settings.UI.Lib.Utilities;
+using System;
+
+namespace CommonLibTest
+{
+    [TestClass]
+    public class HelperTest
+    {
+        public static void TestStringIsSmaller(string v1, string v2)
+        {
+            var res = Helper.CompareVersions(v1, v2);
+            Assert.IsTrue(res < 0);
+        }
+
+
+        public static void TestStringsAreEqual(string v1, string v2)
+        {
+            var res = Helper.CompareVersions(v1, v2);
+            Assert.IsTrue(res == 0);
+        }
+
+        [TestMethod]
+        public void Helper_CompareVersions_ShouldBeEqual_WhenSuccessful()

```

Comment:
the Helper_ suffix is not necessary since you're already testing the Helper class.